### PR TITLE
Add full APM/DBM mode for SQL Server and .Net

### PR DESCRIPTION
**Pull request by nenadnoveljic**
### What does this PR do? What is the motivation?

Adding full APM/DBM mode for SQL Server and .Net.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by lu-zhengda — APPROVED** [review]

**Review by vandonr — APPROVED** [review]
this relates to https://github.com/DataDog/dd-trace-dotnet/pull/5859 btw

File: content/en/database_monitoring/connect_dbm_and_apm.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ APM tracer integrations support a *Propagation Mode*, which controls the amount
 
 \* Full propagation mode on Aurora MySQL requires version 3.
 
-\*\* SQL Server only supports full mode with the Java tracer.
+\*\* SQL Server only supports full mode with the Java and .Net tracer.
```
Comment (alai97) [inline content/en/database_monitoring/connect_dbm_and_apm.md:38]:
```suggestion
\*\* SQL Server only supports full mode with the Java and .NET tracers.
```

File: content/en/database_monitoring/connect_dbm_and_apm.md [hunk 2]
```diff
@@ -78,6 +78,7 @@ APM tracer integrations support a *Propagation Mode*, which controls the amount
 - Prerequisites:
   - Agent version 7.55.0 or greater
   - Java tracer version 1.39.0 or greater
+  - .Net tracer version 3.3 or greater
```
Comment (alai97) [inline content/en/database_monitoring/connect_dbm_and_apm.md:81]:
```suggestion
  - .NET tracer version 3.3 or greater
```

File: content/en/database_monitoring/connect_dbm_and_apm.md [hunk 3]
```diff
@@ -300,7 +301,7 @@ Ensure that you are using a supported client library. For example, `Npgsql`.
 
 Enable the database monitoring propagation feature by setting the following environment variable:
    - For Postgres and MySQL: `DD_DBM_PROPAGATION_MODE=full`
-   - For SQL Server: `DD_DBM_PROPAGATION_MODE=service` or `DD_DBM_PROPAGATION_MODE=full` with Java tracer
+   - For SQL Server: `DD_DBM_PROPAGATION_MODE=service` or `DD_DBM_PROPAGATION_MODE=full` with Java and .Net tracer
```
Comment (alai97) [inline content/en/database_monitoring/connect_dbm_and_apm.md:304]:
```suggestion
   - For SQL Server: `DD_DBM_PROPAGATION_MODE=service` or `DD_DBM_PROPAGATION_MODE=full` with Java and .NET tracers
```

**Review by alai97 — COMMENTED** [review]
Looks good, small capitalization nit.

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**